src/createSignRequest.js

```diff
@@ -0,0 +1,59 @@
+'use strict'
+const constructSignatureString = require('./constructSignatureString')
+const constructDigestString = require('./constructDigestString')
+
+const createSignRequest = (creationOptions) => {
+  const defaultOptions = {
+    constructSignatureString,
+    constructDigestString,
+    signedHeaders: ['(request-target)', '(created)'],
```

Comment:
We should probably just require the header names to passed in without the enclosing brackets.

Reply:
It really depends if we see this `signedheaders` array as a representation of the `headers` field within the signature header or as a more generic object. In the `headers` field these values must have the brackets as part of the string.

For now, I added compatibility with both formats.